src/src_V2/modules/manager/financial/components/payables/Payables.jsx

```diff
@@ -0,0 +1,287 @@
+import { useState } from 'react';
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+  TableSortLabel,
+  Paper,
+  Checkbox,
+} from '@mui/material';
+import { visuallyHidden } from '@mui/utils';
+
+import { Container } from '../../../../../components';
+import NoData from '../../../../../components/no-data/NoData';
+import Layout from '../../../../../components/layout/Layout';
```

Comment:
I think it will beter to have index.js in components and export all from there, it'll enable us to import by name with one import statment. How do you think?